api/src/main/java/org/pmiops/workbench/rdr/RdrMapper.java

```diff
@@ -0,0 +1,118 @@
+package org.pmiops.workbench.rdr;
+
+import java.sql.Timestamp;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.model.SpecificPopulationEnum;
+import org.pmiops.workbench.model.WorkspaceActiveStatus;
+import org.pmiops.workbench.rdr.model.RdrWorkspace;
+import org.pmiops.workbench.rdr.model.RdrWorkspaceDemographic;
+import org.pmiops.workbench.utils.mappers.MapStructConfig;
+
+@Mapper(config = MapStructConfig.class)
+public interface RdrMapper {
+  @Mapping(target = "modifiedTime", source = "lastModifiedTime")
```

Comment:
It's better if we can order them by `source, target`, and then use a string utility to alphabetize them.